src/llfs/page_cache_slot.cpp

```diff
@@ -116,7 +116,7 @@ auto PageCacheSlot::acquire_pin(PageId key, bool ignore_key) noexcept -> PinnedR
 {
   const auto old_state = this->state_.fetch_add(kPinCountDelta, std::memory_order_acquire);
   const auto new_state = old_state + kPinCountDelta;
-  const bool newly_pinned = !this->is_pinned(old_state);
+  const bool newly_pinned = !Self::is_pinned(old_state);
```

Comment:
Putting this comment here because I'm not sure where else to put it. There are a couple other places that I think could als be changed to use `Self` instead of `this` (unless those places were left like that intentionally). I noticed this in `release_pin` line 201, and `evict_if_key_equals` line 247. There are other places that do `this->is_valid()` like in `fill`, for example, but not sure if that's intentional or not.

Reply:
Not intentional; for `static` member functions, `Self::` is preferred.  I'll hunt those down and fix them too!  Thanks